scripts/antlr4/Cypher.g4.copy

```diff
@@ -787,6 +822,7 @@ kU_NonReservedKeywords
         | EXPORT
         | DATABASE
         | USE
+        | START
```

Comment:
I added START as a non-reserved keyword since some of our existing test cases use start as a name.

> But I would add all new lexer rules as a reserved keyword in this PR.

Did you mean non-reserved keyword?

Reply:
Yes add them as non-reserved keyword